### ISSUE TITLE
Display issue when in light mode

### ISSUE THREAD

**Issue by gnaneshreddyy** [mohitjoer/freelance-web#24]
🐞 Issue Description

The "Performance Features" heading isn’t visible in light mode, likely due to insufficient color contrast.
<img width="1236" height="647" alt="Image" src="https://github.com/user-attachments/assets/37fe06eb-adf3-40ca-a295-de0c7b04cb5a" />

There’s also a background rendering issue in the "Platform Features" section, as shown in the screenshots.

<img width="1756" height="891" alt="Image" src="https://github.com/user-attachments/assets/294eacea-7801-4c09-955c-4a1f7d6995b4" />
I’d love to work on fixing these visual issues if the maintainers could assign me. It would also be great if this issue could be tagged with hacktoberfest.
